draft-ietf-gnap-core-protocol.md

```diff
@@ -851,9 +851,9 @@ identifier (string)
     For example, a patient identifier for a medical API or
     a bank account number for a financial API.
 
-The following non-normative example shows the use of both common
-and API-specific fields as part of two different access `type` 
-values. 
+The following non-normative example is asking for three kinds of access (read, write, delete) to each of
+two different locations and two different data types (metadata, images) for a single access token 
```

Comment:
A comment to `locations`: the definition above is more vague than needed, the word "typically" is a red flag. Why not say that `locations` is a list of URIs that the RC can use to access Resource Servers?

Reply:
The reason I've initially held back on defining `locations` as URIs is that ultimately it's up to the RS to recognize and enforce this, so it might not be a URI that the client accesses, it could be an identifier for the RS known by the AS/RS coupling. We could make this more strict now and see if there's need in practice for loosening up this field or defining a new field, but that should be a separate issue (not the topic of this PR).

Reply:
Raised as a new issue in #142.